astro/src/content/docs/apis/_user-search-request-parameters.mdx

```diff
@@ -11,21 +11,20 @@ import SearchPreprocessingWarning from "src/content/docs/_shared/_search-preproc
 
 <APIBlock>
 
-
-  <APIField name={props.parameter_prefix + 'accurateTotal'} type="Boolean" optional defaults="false" since="1.24.0">
-    Set this value equal to `true` to receive an accurate hit count on the API response.
-
-    By default the search engine will limit the hit count to 10,000 users. This means that even if your query may match more than 10,000 users, the returned total count will be 10,000. This is adequate for many use cases such as pagination and general purpose queries. If you are looking for an accurate user count that can exceed 10,000 matches, you will want to set this value equal to `true`.
-  </APIField>
-
-  <APIField name={props.parameter_prefix + 'expand'} type="Array<String>" optional defaults="[memberships, registrations]" since="1.48.0">
+  <APIField name='expand' type="Array<String>" optional defaults="[memberships, registrations]" since="1.48.0">
     This parameter allows you to optionally remove the `memberships` and `registrations` from the API response. Removing these fields from the response may improve performance on large search requests, specifically when you are managing application roles through User Registrations and Group Memberships.
 
     For backwards compatibility, the default behavior will be to return both `memberships` and `registrations`.
 
     To request only the `registrations` but omit the `memberships` from the response, provide a value of `[registrations]`. To omit both the `memberships` and `registrations` from the response, provide a value of `[]`.
   </APIField>
 
+  <APIField name={props.parameter_prefix + 'accurateTotal'} type="Boolean" optional defaults="false" since="1.24.0">
```

Comment:
So for the `GET` `accurateTotal` should be ahead of `expand`, but for the `POST` it should be behind it. Right now the doc in this PR is correct for the `POST` use case.

Seem attached images.

**POST**
<img width="673" alt="Screenshot 2024-08-26 at 11 23 42 AM" src="https://github.com/user-attachments/assets/d2de5d2f-1013-423f-b54f-2110ef202800">

**GET**
<img width="513" alt="Screenshot 2024-08-26 at 11 23 36 AM" src="https://github.com/user-attachments/assets/792a6695-b4ab-4ce2-b9dc-628717b488ee">

Seems like you need an if statement on whether or not `props.parameter_prefix` is set.